JournalCoreData/View Controllers/EntriesTableViewController.swift

```diff
@@ -13,8 +13,8 @@ class EntriesTableViewController: UITableViewController, NSFetchedResultsControl
 
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
-        
-        tableView.reloadData()
+        entryController.fetchEntriesFromServer()
```

Comment:
These two lines are not needed since in your model controller in the ini you are calling already "fetchEntriesFromServer" that gets called once this page is loaded.